media-libs/noise-suppression-for-voice/noise-suppression-for-voice-1.03.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A real-time noise suppression plugin for voice"
+HOMEPAGE="https://github.com/werman/noise-suppression-for-voice"
+
+if [ "${PV}" = "9999" ]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/werman/noise-suppression-for-voice.git"
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/werman/noise-suppression-for-voice/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="+ladspa lv2 vst vst3"
+
+COMMON_DEPEND="x11-libs/libX11 x11-libs/libXcursor x11-libs/libXinerama x11-libs/libXrandr"
+DEPEND="
+	lv2? ( ${COMMON_DEPEND} )
+	vst? ( ${COMMON_DEPEND} )
+	vst3? ( ${COMMON_DEPEND} )
```

Comment:
I'm also now wondering if we should just always-enable all of these. They each have the same set of deps.

Should they be under one USE flag instead, or always enabled?

Reply:
It's fine to keep my as maintainer.  I don't know if the proxy maintainers project wants another package, but I am fine proxying this one myself.

Reply:
Well the dependencies here are dependencies of the JUCE library that `noise-suppression-for-voice` vendors, and the LADSPA plugin does not use JUCE. We could merge lv2+vst+vst3 plugins into one USE flag, not sure what that would be called though.

Reply:
I guess it's fine as-is.